test/everypolitician/popolo/person_test.rb

```diff
@@ -163,4 +163,19 @@ def test_persons_subtraction
     just_person_1 = Everypolitician::Popolo::People.new([person1])
     assert_equal [Everypolitician::Popolo::Person.new(person2)], all_people - just_person_1
   end
+
+  def test_honorific_prefix
+    person1 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob', honorific_prefix: 'Dr')
+    person2 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob')
+    assert_equal 'Dr', person1.honorific_prefix
+    assert_nil person2.honorific_prefix
+  end
+
+  def test_honorific_suffix
+    person1 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob', honorific_suffix: 'PhD')
+    person2 = Everypolitician::Popolo::Person.new(id: '123', name: 'Bob')
+    assert_equal 'PhD', person1.honorific_suffix
+    assert_nil person2.honorific_prefix
```

Comment:
this test will certainly pass OK, but I'm assuming you actually _meant_ to test that person2 doesn't have a an honorific **suffix** rather than prefix?